src/go/rpk/pkg/cli/cmd/cluster/maintenance/enable.go

```diff
@@ -67,16 +75,34 @@ node exists that is already in maintenance mode then an error will be returned.
 			}
 
 			out.MaybeDie(err, "error enabling maintenance mode for node %d: %v", nodeID, err)
-			fmt.Printf("Successfully enabled maintenance mode for node %d\n", nodeID)
+
+			retries := 3
+			for {
+				b, err := client.Broker(cmd.Context(), nodeID)
+				if err != nil {
+					if retries <= 0 {
+						out.Die("Exiting.. error retrieving broker status after emitting the request.")
+					}
+					retries--
+					time.Sleep(time.Second * 2)
+					continue
+				}
```

Comment:
I see the point of checking that the broker exist before calling the maintenance endpoint. But not sure if we should wait for the node to be Draining, I think that's the purpose of the `--wait` flag.

My idea of a flow will be:
- Call client.Broker to see if it's alive.
  - dead node? exit.
- If it's alive, issue the maintenance request:
  - --wait? wait for the node to be drained and the maintenance to be finished.
  - no flag? Exit and guide the user to use `rpk maintenance status`
  
wdyt? 

cc: @twmb

Reply:
My worry is that the wait time could be longer for some users that expect this to be a fast request.

However, I don't know how fast a node can change from !!Draining to Draining. Maybe that's fast and I'm being paranoid. :sweat_smile:

Reply:
Thanks for the idea! To minimize the UX change, your flow looks good to me.